pinot-query-runtime/src/test/java/org/apache/pinot/query/mailbox/channel/MailboxContentObserverTest.java

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.query.mailbox.channel;
+
+import com.google.protobuf.ByteString;
+import io.grpc.stub.StreamObserver;
+import org.apache.pinot.common.proto.Mailbox.MailboxContent;
+import org.apache.pinot.common.proto.Mailbox.MailboxStatus;
+import org.apache.pinot.query.mailbox.MailboxService;
+import org.apache.pinot.query.mailbox.ReceivingMailbox;
+import org.mockito.Mockito;
+import org.testng.annotations.Test;
+
+import static org.mockito.Mockito.*;
+
+public class MailboxContentObserverTest {
+
+  private static final String TEST_MAILBOX_ID = "test-mailbox-id";
+  @Test
+  public void testMailboxContentObserverInitializationAndFetch() {
+    MailboxService mailboxService = mock(MailboxService.class);
+    ReceivingMailbox receivingMailbox = mock(ReceivingMailbox.class);
+    when(mailboxService.getReceivingMailbox(TEST_MAILBOX_ID)).thenReturn(receivingMailbox);
+    StreamObserver<MailboxStatus> mockStatusObserver = mock(StreamObserver.class);
+
```

Comment:
nit / opinionated: I think using empty lines is an anti-pattern since everyone has different opinions about how they view blocks of code and overtime the code can kinda look very inconsistent. It also interrupts the flow of the reader a bit. My opinion there is to prefer single-line comments rather than empty lines. But I'll leave it up to you